### ISSUE TITLE
Implement Leave Of Absence

### ISSUE THREAD

**Issue by liam-middlebrook** [ComputerScienceHouse/conditional#8]
https://github.com/ComputerScienceHouse/Constitution/commit/2904d74aac18e0f5348c7e38384fe86b7c8db87a

We may want to implement a way for members to be exempt from quorum requirements, or calculate quorum minus a certain number of people. We don't need this explicitly because the Evaluations director can just excuse members who are under a leave of absence manually from House Meetings.

**Comment by liam-middlebrook** [comment]
I think it might be a better decision to leave this as an excused absence for House Meetings. I don't see a good !/$ ratio with implementing this

**Comment by stevenmirabito** [comment]
@liam-middlebrook I feel this could be implemented pretty simply as a checkbox in member management, then we can just automatically excuse them from house meetings.

**Comment by mbillow** [comment]
I was talking with @liam-middlebrook about this the other night and I realized that the effects of Leave of Absence basically put a member in the same status as an Alumni in Good Standing. We should have a dropdown of possible statuses for members and below is what should happen.


Active (Dues Paying): Added to Active LDAP Group
Leave of Absence: Remove from Active LDAP Group, Track Status in DB
Alumni: Remove from Active LDAP Group


Basically, we make them an alumni in LDAP's eyes and then we just keep a table of active LOAs.
